### PR TITLE
CDS-3 Add mobile friendly nav menu

### PR DESCRIPTION
**Pull request by forkball**
Changlog:
- Add mobile friendly drop down 
- Add conditional to hide nav once a certain scroll percentage has occured
- Make mobile dropdown full screen

### CODE REVIEW

**Review by Liannus — APPROVED** [review]
Good base. Does need styling, I would also recommend using a CSS based hamburger menu of some kind so it visually changes when the menu is open. Also since this is mobile only you could just make the menu full screen instead of a small dropdown.

File: src/components/MenuIcon/MenuIcon.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+svg {
+    color: $color-off-white;
+    svg:focus {
+        transition: opacity 0.2s ease-in-out;
+        opacity: 0.65;
+    }
+}
+
+ul {
+    background-color: $color-near-black;
```
Comment (Liannus) [inline src/components/MenuIcon/MenuIcon.scss]: Actually just noticed, I believe the footer is sharing one of these components and I think this is the colour causing it to look different.

**Review by Liannus — CHANGES_REQUESTED** [review]
Small fix.

File: src/components/MenuIcon/MenuIcon.scss (hunk 1 above)
Comment (forkball) [reply src/components/MenuIcon/MenuIcon.scss]: Not exactly sure what component you're talking about... Are you saying footer has a ul styling which is messing up color?

Comment (Liannus) [reply src/components/MenuIcon/MenuIcon.scss]: Yeah it was changing the footer colour.

**Review by Liannus — APPROVED** [review]
Looks good.

**Pull request merged**